zkllvm.nix

```diff
@@ -0,0 +1,50 @@
+{ lib,
+  stdenv,
+  src_repo,
+  pkgs,
+  ninja,
+  pkg-config,
+  cmake,
+  boost183,
```

Comment:
It should be like this. Nix is wired

Reply:
Well, you can do it differently. If you don't want to pass boost183 through params, you can only do:

```
  boost_lib ? pkgs.boost183,
```

Not 100% sure it will work, but it should.

Reply:
it's the same in crypto3, I guess, there should be a reason in implementing this way?